sample/src/main/kotlin/com/uber/autodispose/sample/ArchComponentViewModel.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2017. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.uber.autodispose.sample
+
+import android.arch.lifecycle.ViewModel
+import io.reactivex.Observable
+import java.util.concurrent.TimeUnit
+
+/**
+ * Demo Architecture Component ViewModel. The ViewModel
+ * will expose your Rx stream which should be observed
+ * by the view.
+ */
+class ArchComponentViewModel: ViewModel() {
+
+  /**
+   * Simple observable that emits a sequence of
+   * numbers ever second.
+   */
+  fun observable(): Observable<Long> {
+    return Observable.interval(1, TimeUnit.SECONDS)
```

Comment:
Should this be something hot+cached or otherwise a bit more realistic? Thinking something like a network request to download something that completes in the background and is cached if it finishes the load even while not subscribed to. Could use something like [RxReplayingShare](https://github.com/JakeWharton/RxReplayingShare) under the hood to manage the caching. What do you think?

Reply:
Yeah! Definitely need something more realistic to show that it's something comparable to LiveData. I'd be happy to integrate something like RxReplayingShare. Do you think we should have a network request in the sample or should it just be the same sequence Observable backed by caching so that when you rotate, you kinda resume the sequence?

Reply:
Maybe rendering an expensive bitmap from res/raw and showing the result in an image view?

Reply:
Have included a sample that loads an image from `res/raw`. Have also included an `ImageRepository` to load the image, instead of doing it all in the ViewModel. This will model a real world example more closely.